aiaccel/torch/lightning/callback.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import lightning.pytorch as pl
```

Comment:
```suggestion
import lightning
```
確かplはdeprecatedでlightningだったような…